src/main/java/io/kestra/plugin/cloudquery/CloudQueryCLI.java

```diff
@@ -0,0 +1,101 @@
+)
+@Plugin(
+    examples = {
+        @Example(
+            title = "Migrate CloudQuery destination schemas",
+            code = {
+                "tasks:",
+                "  - id: wdir",
+                "    type: io.kestra.core.tasks.flows.WorkingDirectory",
+                "    tasks:",
+                "      - id: config",
+                "        type: io.kestra.core.tasks.storages.LocalFiles",
+                "        inputs:",
+                "          config.yml: |",
+                "            kind: source",
+                "            spec:",
+                "              name: aws",
+                "              path: cloudquery/aws",
+                "              version: \"v22.4.0\"",
+                "              tables: [\"aws_s3*\", \"aws_ec2*\", \"aws_ecs*\", \"aws_iam*\", \"aws_glue*\", \"aws_dynamodb*\"]",
+                "              destinations: [\"postgresql\"]",
+                "              spec:",
+                "            ---",
+                "            kind: destination",
+                "            spec:",
+                "              name: \"postgresql\"",
+                "              version: \"v5.0.3\"",
+                "              path: \"cloudquery/postgresql\"",
+                "              write_mode: \"overwrite-delete-stale\"",
+                "              spec:",
+                "                connection_string: ${PG_CONNECTION_STRING}",
+                "",
+                "      - id: cloudQuery",
+                "        type: io.kestra.plugin.cloudquery.CloudQueryCLI",
+                "        env:",
+                "          AWS_ACCESS_KEY_ID: \"{{ secret('AWS_ACCESS_KEY_ID') }}\"",
+                "          AWS_SECRET_ACCESS_KEY: \"{{ secret('AWS_SECRET_ACCESS_KEY') }}\"",
+                "          AWS_DEFAULT_REGION: \"{{ secret('AWS_DEFAULT_REGION') }}\"    ",
+                "          PG_CONNECTION_STRING: \"postgresql://postgres:{{secret('DB_PASSWORD')}}@host.docker.internal:5432/demo?sslmode=disable\"",
+                "        commands:",
+                "        - cloudquery migrate config.yml --log-console"
+            }
+        ),
+    }
+)
+public class CloudQueryCLI extends AbstractCloudQueryCommand implements RunnableTask<ScriptOutput> {
+
+    @Schema(
+        title = "List of commands to run"
+    )
+    @PluginProperty(dynamic = true)
+    @NotNull
+    @NotEmpty
+    protected List<String> commands;
+
+    @Override
+    public ScriptOutput run(RunContext runContext) throws Exception {
+        CommandsWrapper commands = new CommandsWrapper(runContext)
+            .withWarningOnStdErr(true)
+            .withRunnerType(RunnerType.DOCKER)
+            .withDockerOptions(injectDefaults(getDocker()))
+            .withCommands(
+                ScriptService.scriptCommands(
+                    List.of("/bin/sh", "-c"),
+                    List.of("export PATH=\"/app:$PATH\""),// add cloudquery path to the path
```

Comment:
I'm really not a fan, the image have the binary as the command maybe the list of cloud query commands must be cloud query commands and not shell commands?
Another way would be to create an env var instead of using export.